### PR TITLE
fix resize for segmentation masks without channel dimension

### PR DESCRIPTION
**Pull request by pmeier**
Follow-up to #6574.

```py
import torch

from torchvision.prototype import features
from torchvision.prototype.transforms import functional as F

size = (16, 31)

segmentation_mask = features.SegmentationMask(torch.randint(0, 10, size))

print(segmentation_mask.shape, F.resize(segmentation_mask, size).shape)

```

```
torch.Size([16, 31]) torch.Size([1, 16, 31])
```

This was not detected in #6574 since kernel works without adding an explicit channel dimension, but it doesn't remove it again. This happens because

https://github.com/pytorch/vision/blob/b4686f2b7409d1783dfbb951492cd59bfed08bce/torchvision/prototype/transforms/functional/_geometry.py#L106

returns `1` for the number of channels if we have no channel dimension:

https://github.com/pytorch/vision/blob/b4686f2b7409d1783dfbb951492cd59bfed08bce/torchvision/transforms/functional_tensor.py#L24-L26

### CODE REVIEW

File: torchvision/transforms/functional_tensor.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def hflip(img: Tensor) -> Tensor:
 def crop(img: Tensor, top: int, left: int, height: int, width: int) -> Tensor:
     _assert_image_tensor(img)
 
-    _, h, w = get_dimensions(img)
+    w, h = get_image_size(img)
```
Comment (pmeier) [inline torchvision/transforms/functional_tensor.py:135]: I found this while debugging `get_dimensions`. Since the number of channels is not used, we can get away with using `get_image_size`. Took me a while to figure out that it returns width first :roll_eyes: 

LMK if you don't want this change in this PR.

**Review by datumbox — APPROVED** [review]
LGTM, thanks!

File: torchvision/transforms/functional_tensor.py (hunk 1 above)
Comment (datumbox) [reply torchvision/transforms/functional_tensor.py]: I will be removing the use of `get_image_size()` from stable `ElasticTransform` on a separate PR.
